packages/seacas/libraries/ioss/src/visualization/exodus/Iovs_exodus_DatabaseIO.C

```diff
@@ -567,6 +565,73 @@ namespace Iovs_exodus {
+  int64_t DatabaseIO::put_field_internal(const Ioss::EdgeBlock * /*eb*/, const Ioss::Field &field,
+                                         void * /*data*/, size_t data_size) const
+  {
+    return get_num_to_get(field, data_size);
+  }
+  int64_t DatabaseIO::put_field_internal(const Ioss::FaceBlock * /*fb*/, const Ioss::Field &field,
+                                         void * /*data*/, size_t data_size) const
+  {
+    return get_num_to_get(field, data_size);
+  }
+  int64_t DatabaseIO::put_field_internal(const Ioss::SideBlock * /*sb*/, const Ioss::Field &field,
+                                         void * /*data*/, size_t data_size) const
+  {
+    return get_num_to_get(field, data_size);
+  }
+  int64_t DatabaseIO::put_field_internal(const Ioss::NodeSet * /*ns*/, const Ioss::Field &field,
+                                         void * /*data*/, size_t data_size) const
+  {
+    return get_num_to_get(field, data_size);
+  }
+  int64_t DatabaseIO::put_field_internal(const Ioss::EdgeSet * /*es*/, const Ioss::Field &field,
+                                         void * /*data*/, size_t data_size) const
+  {
+    return get_num_to_get(field, data_size);
+  }
+  int64_t DatabaseIO::put_field_internal(const Ioss::FaceSet * /*fs*/, const Ioss::Field &field,
+                                         void * /*data*/, size_t data_size) const
+  {
+    return get_num_to_get(field, data_size);
+  }
+  int64_t DatabaseIO::put_field_internal(const Ioss::ElementSet * /*es*/, const Ioss::Field &field,
+                                         void * /*data*/, size_t data_size) const
+  {
+    return get_num_to_get(field, data_size);
+  }
+  int64_t DatabaseIO::put_field_internal(const Ioss::SideSet * /*ss*/, const Ioss::Field &field,
+                                         void * /*data*/, size_t data_size) const
+  {
+    return get_num_to_get(field, data_size);
+  }
+  int64_t DatabaseIO::put_field_internal(const Ioss::CommSet * /*cs*/, const Ioss::Field &field,
+                                         void * /*data*/, size_t data_size) const
+  {
+    return get_num_to_get(field, data_size);
+  }
+  int64_t DatabaseIO::put_field_internal(const Ioss::StructuredBlock * /*sb*/,
+                                         const Ioss::Field &field, void * /*data*/,
+                                         size_t             data_size) const
+  {
+    return get_num_to_get(field, data_size);
+  }
+  int64_t DatabaseIO::put_field_internal(const Ioss::Assembly * /*as*/, const Ioss::Field &field,
+                                         void * /*data*/, size_t data_size) const
+  {
+    return get_num_to_get(field, data_size);
+  }
+  int64_t DatabaseIO::put_field_internal(const Ioss::Blob * /*bl*/, const Ioss::Field &field,
+                                         void * /*data*/, size_t data_size) const
+  {
+    return get_num_to_get(field, data_size);
+  }
+  size_t DatabaseIO::get_num_to_get(const Ioss::Field &field, size_t data_size) const
```

Comment:
Could probably put this as a `namespace{}` function in the current file.  Does not need to be  a class member function.

Reply:
But not needed for this PR.   Could modify later if wanted.

Reply:
Okay. Will push a change for this.